src/platformdirs/unix.py

```diff
@@ -153,12 +153,15 @@ def user_runtime_dir(self) -> str:
          ``$XDG_RUNTIME_DIR/$appname/$version``.
 
          For FreeBSD/OpenBSD/NetBSD, it would return ``/var/run/user/$(id -u)/$appname/$version`` if
-         ``$XDG_RUNTIME_DIR`` is not set.
+         exists, otherwise ``/tmp/runtime-$(id -u)/$appname/$version``, if``$XDG_RUNTIME_DIR``
+         is not set.
         """
         path = os.environ.get("XDG_RUNTIME_DIR", "")
         if not path.strip():
             if sys.platform.startswith(("freebsd", "openbsd", "netbsd")):
                 path = f"/var/run/user/{getuid()}"
+                if not Path(path).exists():
+                    path = f"/tmp/runtime-{getuid()}"
```

Comment:
From a quick Google search, Ruff doesn't seem to give us an in-depth reasoning behind this [rule](https://beta.ruff.rs/docs/rules/#S108). This StackOverflow [answer](https://stackoverflow.com/a/65333940) may provide the reason, where creating a predictable temp directory path could be susceptible to a "time of check, time of use attack (TOCTOU)".

I'm not familiar with this path, but I do see it used in a couple Unix applications from what I could find. If it won't be a problem with the maintainers, you could ignore this rule by adding `# noqa: S108` to this line.